libs/core/src/lib/select/public_api.ts

```diff
@@ -1,5 +1,5 @@
 export * from './select.module';
 export * from './select.component';
-export * from './select-proxy.service';
+export * from './option/option.component';
```

Comment:
Doubled `export * from './option/option.component';` export

Reply:
Too many updates with this change